ingestion_tools/scripts/tests/s3_import/test_annotations.py

```diff
@@ -948,3 +963,75 @@ def test_ingest_instance_point_data(
 
         # Check id
         assert exp_point["instance_id"] == point["instance_id"], f"Incorrect id for {case['case']}"
+
+
+@pytest.mark.parametrize(
+    "glob_string,file_format",
+    [
+        ("annotations/triangular_mesh.stl", "stl"),
+        ("annotations/triangular_mesh.glb", "glb"),
+        ("annotations/triangular_mesh.vtk", "vtk"),
+        ("annotations/triangular_mesh.obj", "obj"),
+        ("annotations/triangular_mesh.obj", "obj"),
+    ],
+)
+def test_ingest_triangular_mesh(
+    glob_string,
+    file_format,
+    tomo_importer: TomogramImporter,
+    dataset_config_local: DepositionImportConfig,
+    local_test_data_dir: str,
+):
+    # Arrange
+    dataset_config_local._set_object_configs(
+        "annotation",
+        [
+            {
+                "metadata": default_anno_metadata,
+                "sources": [
+                    {
+                        "TriangularMesh": {
+                            "file_format": file_format,
+                            "glob_string": glob_string,
+                            "is_visualization_default": False,
+                        },
+                    },
+                ],
+            },
+        ],
+    )
+
+    # Action
+    anno = TriangularMeshAnnotation(
+        config=dataset_config_local,
+        metadata=default_anno_metadata,
+        path=os.path.join(local_test_data_dir, "input_bucket/20002", glob_string),
+        parents={"tomogram": tomo_importer, **tomo_importer.parents},
+        file_format=file_format,
+        identifier=100,
+    )
+    anno.import_item()
+    anno.import_metadata()
+
+    # Assert
+    # verify local_metadata
+    path = "dataset1/run1/Tomograms/VoxelSpacing10.000/Annotations/100-some_protein-1.0_"
+    # if name:
+    #     path += f"{index}"
+    path +="triangularmesh.glb"
+    expected_local_metadata = {
+        "object_count": 1,
+        "files": [
+            {
+                "format": "glb",
+                "path": path,
+                "shape": "TriangularMesh",
+                "is_visualization_default": False,
+            },
+        ],
+    }
+    assert anno.local_metadata == expected_local_metadata
+
+    # load the new mesh file
+    anno_file = anno.get_output_filename(anno.get_output_path())
+    trimesh.load(anno_file)
```

Comment:
What is your thought on adding a regression check here by having an expected mesh, that we can compare to the output of this line?